version.bzl

```diff
@@ -1,3 +1,3 @@
 # NOTE: If you change the format of this line, you must change the bash command
 # in /scripts/publish to extract the version string correctly.
-DF_VERSION = "1.5.1"
+DF_VERSION = "1.5.2"
```

Comment:
i don't think we need this? your changes aren't in `core`, so nobody needs to update their package.json.